CSSfiles/main.scss

```diff
@@ -14,6 +14,107 @@
     font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
   }
 
+  // the pop-up section
+
+  .the-pop-upD {
+    display: none;
+  }
+
+  .the-pop-up {
```

Comment:
- [ ] Fix your background images , they are not displaying.
- [ ] I noticed that upon opening the popup another popup without styling shows on the top left of the webpage. Kindly fix this issue. 
- [ ] The background color of the popup needs to match figma design. 

![Ademomla's Portfolio](https://user-images.githubusercontent.com/60618877/180364439-9f145c9e-01ca-4207-83c6-5ccad8540201.gif)

Kindly fix the issues mentioned and the project will be ready to go.